Masters.Api/Filters/ResourcePerformanceFilter.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.AspNetCore.Mvc.Filters;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Masters.Api.Filters
+{
+    public class ResourcePerformanceFilter : IResourceFilter
+    {
+        private readonly Stopwatch stopwatch = new Stopwatch();
```

Comment:
Hi sir, current implementation is prone to race condition problem. I recommend storing the stopwatch in the http context items. You may refer to the Tip page of session 7 ppt